test/e2e/providers/vsphere/vsphere.go

```diff
@@ -11,14 +11,20 @@ import (
 	mapi "github.com/openshift/api/machine/v1beta1"
 	"github.com/pkg/errors"
 	core "k8s.io/api/core/v1"
+	storage "k8s.io/api/storage/v1"
+	k8sapierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 	client "k8s.io/client-go/kubernetes"
 
 	"github.com/openshift/windows-machine-config-operator/test/e2e/clusterinfo"
 	"github.com/openshift/windows-machine-config-operator/test/e2e/providers/machineset"
 )
 
-const defaultCredentialsSecretName = "vsphere-cloud-credentials"
+const (
+	defaultCredentialsSecretName = "vsphere-cloud-credentials"
+	storageClassName             = "e2e"
```

Comment:
The same constant was introduced in the previous commit. Is it possible to consolidate?

Reply:
Unfortunately not with the way the packages are structured

Reply:
Can't you introduce a constant in a higher level package?

Reply:
I could put it in the clusterinfo package or create a new one.
That will make cherry-picking the in-tree vsphere test to release-4.13 more difficult.